src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $CardWidth;
+  padding: $CardWidth * 0.01 * 16 $CardWidth * 0.01 * 8 $CardWidth * 0.01 * 8;
+  border: 1px solid $ColorCardBorder;
+
+  @include transition-options(transform);
+
+  &__image {
+    width: $CardWidth * 0.8;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $FontMedium;
```

Comment:
The variable `$FontMedium` should be written in lowercase letters with hyphens instead of camelCase, as per the code style rules.